public/js/scripts.js

```diff
@@ -554,6 +571,19 @@ function submitQuestion() {
     //sending the data as a json requires this hash form of data(i think, never tried it without it)
     var data = {
         question: text  //'question' is the key and 'text' is the associated data
+function sendfeedback_comment(){
+    var feedback_comment = $("#feedback_commentTextarea").val();
+    $.post("/api/feedback_comment",{feedback_comment:feedback_comment},function(data){
+        $(".filterName").html("feedback submit result:"+ data.message);
+        togglefeedbackForm();
+    })
+}
+
+function submitQuestion() {
```

Comment:
Also this looks like an uncaught merge conflict. This is declared here and then again above on line 567. Strange that git did not catch these.